### DISCUSSION TITLE
Email

### DISCUSSION THREAD

**Discussion by thatswhatsimonsaid** [anhourandaquarter/carmina#20] [General]
Hi all!

For some reason, I keep getting a failed mail delivery when sending you all an email. Could you send me your 
<img width="1440" alt="Screenshot 2024-11-25 at 8 41 56 PM" src="https://github.com/user-attachments/assets/309c05fc-bf17-4f66-98d7-39d2cb86a4f1">
secondary emails?

Simon

**Comment by thatswhatsimonsaid** [comment]
I saw your email, but this also happens when I respond to your email @Emend026

**Reply by Emend026** [reply]
Oh that's weird. Emend026@uw.edu 

Emendnova026@gmail.com is my personal 

@thatswhatsimonsaid

**Comment by thatswhatsimonsaid** [comment]
I won't be able to make it to class today due to a last-minute industry thing, but I will be over zoom if you guys need to chat! Email me if you want me to hop on

**Comment by hhchan94** [comment]
I also cannot make it to the class on time due to the other meeting, so let's chat via Zoom.

**Comment by hhchan94** [comment]
Join Zoom Meeting
https://washington.zoom.us/j/92355316315

Meeting ID: 923 5531 6315

**Comment by infinite0x20** [comment]
Dave's giving a lecture right now, so Liz and I won't be able to join for a bit

**Comment by Emend026** [comment]
Are you guys free to hop on zoom after 5pm? evening meet up

**Comment by hhchan94** [comment]
I can meet at 5

**Comment by infinite0x20** [comment]
I could do 5:30 and after

**Comment by thatswhatsimonsaid** [comment]
Sorry - I have a conference deadline to NeurIPS this Friday so I'll be unavailable. I plan to be in class on Thursday

**Comment by Emend026** [comment]
For the rest of us, would you guys prefer to meet tonight or Thursday after
class?

Elizabeth Nova
PhD Student
Department of Sociology
University of Washington

**Comment by infinite0x20** [comment]
I forgot I have another work-related meeting tonight so I think Thursday might actually be best. In the meantime, these are the tasks that we’ll focus on next (will make new issues and branches for the ones with *):- updating environment.yml *- fixing style for parser.py *- fixing style for test_parser.py *- starting scansion.py helper functionsOn Dec 3, 2024, at 16:15, Elizabeth Nova ***@***.***> wrote:﻿
For the rest of us, would you guys prefer to meet tonight or Thursday after

class?



Elizabeth Nova

PhD Student

Department of Sociology

University of Washington





On Tue, Dec 3, 2024 at 3:39 PM thatswhatsimonsaid ***@***.***>

wrote:
